workbench-core/infrastructure/src/workbenchCognito.ts

```diff
@@ -68,15 +69,20 @@ const userPoolClientDefaults: UserPoolClientOptions = {
   refreshTokenValidity: Duration.days(7)
 };
 
+export interface WorkbenchCognitoUserPoolClientProps {
+  userPoolClientName?: string;
+  authFlows?: AuthFlow;
+  accessTokenValidity?: Duration;
+  idTokenValidity?: Duration;
+  refreshTokenValidity?: Duration;
+}
+
 export interface WorkbenchCognitoProps {
   domainPrefix: string;
   websiteUrls: string[];
   userPoolName?: string;
-  userPoolClientName?: string;
   oidcIdentityProviders?: WorkbenchUserPoolOidcIdentityProvider[];
-  accessTokenValidity?: Duration;
-  idTokenValidity?: Duration;
-  refreshTokenValidity?: Duration;
+  userPoolClients?: WorkbenchCognitoUserPoolClientProps[];
```

Comment:
The web ui client should have a separate field since it is handled differently than the programmatic access clients

Reply:
I'm not sure I understand since this is for CloudFormation template - webUi client is created with 

```
authFlows: {adminUserPassword: false}
```

Reply:
Since the web UI client is used in a different way than the other clients it should be split out so a user doesnt have to remember where the web UI clientId was in the array of clientIds or if they didnt include it at all

Reply:
I would also recommend removing `adminUserPassword: true` from the `userPoolClientDefaults` object and force the user to allow that flow explicitly rather than allow it by default.